src/renderer/components/ft-list-channel/ft-list-channel.scss

```diff
@@ -32,13 +35,6 @@
   }
 }
 
-.infoAndSubscribe {
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
-  inline-size: 100%;
-}
```

Comment:
`.infoAndSubscribe` under `&.list` has no `display: flex;` & `inline-size: 100%;`
Expected?

Reply:
This is a very old PR, so I can't remember for sure, but it seems like I groggily made changes to address the `no-descending-specificity` rule that I can't tell if they're intentional (and crafted to stop things from breaking) or are just mistakes. I'm going to comb through all of these again and check back when I'm done.